cypress/e2e/signUp.cy.js

```diff
@@ -1,12 +1,71 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignUpPage from '../support/pages/signUp.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import faker from 'faker';
 
 describe('Sign Up page', () => {
+  let name;
+  let email;
+  let password;
+  const signUpPage = new SignUpPage();
+
   before(() => {
+    cy.task('db:clear');
+  });
```

Comment:
Consider adding a call to `cy.task('db:seed')` after `cy.task('db:clear')` to ensure there is dummy data to use for testing.

Reply:
It's really good idea, I can try to make this approach after checking of all my previous mistakes.